### ISSUE TITLE
Ask to the developer for another project (app)

### ISSUE THREAD

**Issue by serrq** [v1tzor/TimePlanner#75]
I am convinced of the solidity of the Bitcoin Cash (BCH) project that in recent months has been accepted as currency in many communities of South America and Africa, with a potentially explosive economy.

A project developed on BCH is read.cash that establishes a value for value return to the writer of an interesting article.

This can be both a blog (visibility) and a small gain for a university student who wants to be more independent from parents and read.cash is a good way to do it.

I therefore ask to the developer if he is interested in writing an app dedicated to this blogging platform.

https://read.cash/

Basically it is a writing blog with a built-in BCH wallet to get revenues.

ps: there is no read.cash app in fdroid.
